frontend/src/components/pages/AdminPage.tsx

```diff
@@ -1,35 +1,112 @@
-import * as Routes from "../../constants/Routes";
+import {
+  Flex,
+  Text,
+  Box,
+  Tabs,
+  TabList,
+  Tab,
+  TabPanel,
+  TabPanels,
+} from "@chakra-ui/react";
+import SideBar from "../common/Sidebar";
 
-import Logout from "../auth/Logout";
-import RefreshCredentials from "../auth/RefreshCredentials";
-import ResetPassword from "../auth/ResetPassword";
+import Page from "../../types/PageTypes";
+import AdminUserTable from "../common/AdminUserTable";
+import AddAdminModal from "../common/AddAdminModal";
 
-type ButtonProps = { text: string; path: string };
+const pages: Page[] = [
+  { title: "Assessments", url: "/" },
+  { title: "Database", url: "/" },
+];
 
-const Button = ({ text, path }: ButtonProps) => {
-  const history = useHistory();
-  const navigateTo = () => history.push(path);
-  return (
-    <button className="btn btn-primary" onClick={navigateTo} type="button">
-      {text}
-    </button>
-  );
-};
+const USERS = [
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email:
+      "The quick brown fox jumps over the lazy dog is an English-language pangram—a sentence that contains all of the letters of the English alphabet. Owing to its existence, Chakra was created.",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+  {
+    firstName: "Albert",
+    lastName: "Kuhl",
+    email: "albertkuhl@jumpmath.ca",
+  },
+];
 
 const AdminPage = (): React.ReactElement => {
   return (
-    <div style={{ textAlign: "center", margin: "0px auto" }}>
-      <h1>Admin Page</h1>
-      <div className="btn-group">
-        <Logout />
-        <RefreshCredentials />
-        <ResetPassword />
-        <Button text="Student Page" path={Routes.STUDENT_PAGE} />
-        <Button text="Component Library" path={Routes.COMPONENT_LIBRARY} />
-      </div>
-    </div>
+    <Flex mx="4">
+      <Box p={0} w="20%">
+        <SideBar pages={pages} />
+      </Box>
+      <Flex direction="column" w="100%" pt={20}>
+        <Text
+          textStyle="header4"
+          color="blue.300"
+          style={{ textAlign: "left" }}
+        >
+          Database
+        </Text>
+        <Box textAlign="right" pr={10}>
+          <AddAdminModal />
+        </Box>
+        <Tabs pr={5}>
```

Comment:
could we update the tab text to align with the colors specified by design?

Reply:
I don't think this is possible :(

I tried setting the unselected/selected color. I can make the tab text align with the design BUT the underline/indicator that shows which tab is selected will go away (because they use the same colors for the text and the indicator i believe).

screenshot:
![Screen Shot 2022-11-17 at 3 30 46 PM](https://user-images.githubusercontent.com/32875500/202553418-7f9d0f84-7312-4827-a4a8-9d12811583d0.png)

Reply:
but I have updated the unselected color of text to be the same as figma